mod-source-record-manager-server/src/main/java/org/folio/dao/JobExecutionDaoImpl.java

```diff
@@ -213,10 +213,10 @@ public Future<Boolean> deleteJobExecutionsByIds(List<String> ids, String tenantI
       String query = format(DELETE_BY_IDS_SQL, convertToPsqlStandard(tenantId), TABLE_NAME, ids.stream().collect(Collectors.joining("','")));
       pgClientFactory.createInstance(tenantId).execute(query, promise);
     } catch (Exception e) {
-      LOGGER.error("Error deleting jobExecution by ids {} : Exception : ", ids, e);
+      LOGGER.error("Error deleting jobExecution by ids {}, ", ids, e);
       promise.fail(e);
     }
-    return promise.future().map(updateResult -> updateResult.rowCount() == 1);
+    return promise.future().map(updateResult -> updateResult.rowCount()>0);
```

Comment:
nitpick: could you space this adequately? A spaces surrounding `>`

Reply:
@okolawole-ebsco - New code pushed in, above changes not required, Thanks 🙂.